feature-libs/product-configurator/common/components/configurator-cart-entry-info/configurator-cart-entry-info.component.ts

```diff
@@ -3,13 +3,15 @@ import { FormControl } from '@angular/forms';
 import { OrderEntry } from '@spartacus/core';
 import { CartItemContext } from '@spartacus/storefront';
 import { EMPTY, Observable } from 'rxjs';
+import { CommonConfiguratorUtilsService } from '../../shared/utils/common-configurator-utils.service';
 
 @Component({
   selector: 'cx-configurator-cart-entry-info',
   templateUrl: './configurator-cart-entry-info.component.html',
```

Comment:
better have this `changeDetection: ChangeDetectionStrategy.OnPush,`

Reply:
picked

Reply:
done